dashboard/app/models/pd/regional_partner_contact.rb

```diff
@@ -35,14 +35,18 @@ def send_regional_partner_contact_emails
 
       if regional_partner_program_managers.empty?
         matched_but_no_pms = true
-        Pd::RegionalPartnerContactMailer.unmatched(form, matched_but_no_pms).deliver_now
+        # TODO: When cc supported, send one unmatched email with Jenna cc'ed
+        Pd::RegionalPartnerContactMailer.unmatched(form, 'nimisha@code.org', matched_but_no_pms).deliver_now
+        Pd::RegionalPartnerContactMailer.unmatched(form, 'jenna@code.org', matched_but_no_pms).deliver_now
```

Comment:
I slightly prefer just making the changes inside of `Pd::RegionalPartnerContactMailer.unmatched` - for now, having it send two emails (call `mail()` twice, once for each email), so that this file doesn't have to change at all when cc works.

Reply:
never mind - Tessa told me that `mail` will only be called once here and subsequent calls will be ignored.